tests/index.test.ts

```diff
@@ -1,34 +1,137 @@
-import {argFilter, argSort, mod} from '../src';
+import {argFilter, argSort, bounds, mod, version, uniqueId, uniqueString, resolveIn, randomId, flagId} from '../src';
 
 describe('argFilter', () => {
-  it('evens', () => {
-    expect(argFilter([1, 3, 5, 2, 4, 6, 7, 9, 11], (d) => d % 2 === 0))
-      .toEqual([3, 4, 5]);
-  });
+  it('evens', () => expect(argFilter([1, 3, 5, 2, 4, 6, 7, 9, 11], (d) => d % 2 === 0))
+      .toEqual([3, 4, 5]));
 });
 
-
 describe('argSort', () => {
-  it('simple', () => {
-    expect(argSort(['lizard', 'marsupial', 'cat', 'dolphin'], (a, b) => a.length - b.length))
-      .toEqual([2, 0, 3, 1]);
-  });
+  it('length', () => expect(argSort(['lizard', 'marsupial', 'cat', 'dolphin'], (a, b) => a.length - b.length))
+      .toEqual([2, 0, 3, 1])
+  );
 });
 
-describe('mode', () => {
-  it('+ % +', () => {
-    expect(mod(101, 5)).toEqual(1);
-  });
-  it('- % + (native)', () => {
-    expect(-101 % 5).toEqual(-1);
-  });
-  it('- % +', () => {
-    expect(mod(-101, 5)).toEqual(4);
-  });
-  it('+ % -', () => {
-    expect(mod(101, -5)).toEqual(-4);
-  });
-  it('- % -', () => {
-    expect(mod(-101, -5)).toEqual(-1);
+describe('mod', () => {
+  it('+ % +', () => expect(mod(101, 5)).toEqual(1));
+  it('- % + (native)', () => expect(-101 % 5).toEqual(-1));
+  it('- % +', () => expect(mod(-101, 5)).toEqual(4));
+  it('+ % -', () => expect(mod(101, -5)).toEqual(-4));
+  it('- % -', () => expect(mod(-101, -5)).toEqual(-1));
+});
+
+describe('bounds', () => {
+  /* TODO: This seems odd. For instance, there is already an x and y provided by the DOM,
+  but we give a different meaning to these.  */
+  it('not a DOM object', () => expect(bounds(null)).toEqual({ x: 0, y: 0, w: 0, h: 0 }));
+  // TODO: DOM object
+});
+
+it('version', () => expect(version).toEqual('__VERSION__') );
+// TODO: (1) Doesn't seem right. (2) Does anything actually need it?
+
+describe('uniqueId', () => {
+  it('first time', () => expect(uniqueId()).toEqual(0));
+  it('second time', () => expect(uniqueId()).toEqual(1));
+});
+
+describe('uniqueString', () => {
+  it('first time', () => expect(uniqueString()).toEqual('_default2'));
```

Comment:
Order dependence in the tests is admittedly not a good thing... If there is a way to reset the internal counter between tests, that would be good?